src/requestor/routes/manage.py

```diff
@@ -189,6 +223,14 @@ async def create_request(
             )
         res = request.to_dict()
 
+    await grant_or_revoke_arborist_policy(
```

Comment:
a unit test showing a 200 response when the created request has an "update" status would be nice, even better if we can show that arborist was called... the existing `test_update_request` doesn't show that arborist was called, i wonder if there's a nice way we can do this check

Reply:
Arborist was called in the sense? `grant_user_policy` function being called? Or a request is sent to `arborist`?

Reply:
whichever - grant_user_policy or gen3authz or arborist